### DISCUSSION TITLE
[RFC] Do perpetual hashes regenerate if the media file is moved in a harddisk

### DISCUSSION THREAD

**Discussion by rajatmodi62** [stashapp/stash#3984] [Q&A] [answered]
Hi, 
i was having a small q in stashapp:
Assuming that the perpetual hash, and scene cover are built for a video, 
if i move the media location from one folder to another , 
does the perpetual hash get recomputed again?

if so, is there any way to prevent the recomputation, 
thanks

**Comment by HereIam80** [comment, marked as answer]
Nothing has to be generated again as long as you don't also make any changes to the files themselves.
If you run a scan after moving the files, the paths will get updated during a scan.
Also don not COPY the files as that will make you new entries into your database as both files are still present.
